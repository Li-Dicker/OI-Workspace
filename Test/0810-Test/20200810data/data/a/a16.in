10
198
78826 17216 3000 54015 24132 37746 4062 82962 53174 15097 845 3750 30150 82308 29702 617 29358 45579 86221 61857 45671 57313 93353 7671 66625 9638 93439 51021 84345 81162 29613 63094 75860 30253 82142 75118 60996 53388 90816 43574 22012 36236 91034 75383 33542 65514 87281 32412 26523 91637 97404 84119 29746 29475 58617 33747 71380 48507 91806 49778 30356 90161 41707 45589 11675 92218 14959 41046 69817 24446 73077 83149 31754 76626 32420 29982 31940 78049 3156 63691 64465 97088 66389 13699 65787 46989 21300 41708 83481 55048 3206 864 86440 97582 4484 70125 79666 72241 31879 31726 1696 49582 75143 54016 68542 53597 75904 30124 70889 14718 20418 36186 15367 94693 35179 69793 55976 56220 51705 87004 92224 53104 96871 76730 34044 40875 43203 22759 27427 80190 19243 3937 34227 93478 44423 61042 60954 33638 93855 58135 10647 17792 8649 44153 35021 10936 32760 53520 77665 24985 69310 51610 8336 19041 79373 38494 77771 3274 35640 68117 86179 45689 10856 1461 87425 25390 77507 56556 98510 1308 9477 76591 31226 83757 50461 91496 93929 81941 33901 47946 65805 1332 42735 96053 55565 78383 52712 19100 36896 91143 34761 56125 65689 49186 91365 84457 37202 1561 
48294 27594 35868 32376 34130 13846 13486 12393 11756 10652 17538 4751 28770 5863 40430 23455 17121 42137 49584 29624 38254 1797 2683 27017 504 35546 20840 14812 37990 3482 16140 22497 31439 24158 10412 24068 575 16675 4175 39760 44905 27574 41846 21292 12669 46160 34262 24642 2342 22028 12412 47364 883 37221 13119 33778 36203 24220 35157 523 37937 405 3590 9399 2977 6487 16198 11171 31015 36040 41229 26998 44978 19859 169 24328 46525 20035 41876 28346 37187 26563 17876 31990 39762 26714 30944 29817 16271 49564 20401 35232 36661 17400 44480 17879 28798 37839 28743 86 24458 12156 39846 49817 39946 1849 28693 18177 18290 8021 7785 3701 39938 35230 29772 29549 34575 19072 37589 33935 47609 3145 21383 5348 38624 20160 34020 42936 10649 40514 26707 3420 38148 2449 35552 7363 45338 29927 33693 47880 39359 24808 6131 30592 20973 22728 29155 11280 35332 45885 27209 44280 40257 18629 31240 376 7338 25771 4980 8915 31654 13655 9076 16526 13492 11253 1151 6415 48426 33966 16883 8448 10103 4044 43848 21922 6181 46826 36058 18745 12417 46645 9466 21721 30767 47215 28969 8426 41084 26509 19807 46701 40444 23332 10693 42711 15338 41176 
199
55495 20930 37522 44384 67074 10264 7778 3097 40605 36516 88966 41033 99981 49852 23975 1485 97777 65802 35247 10685 48258 32413 22484 62216 89776 55247 61058 11249 98255 25732 76830 93179 35854 47627 16643 98351 60142 99590 31305 90456 80914 14365 29916 34018 27935 41179 59275 52072 32840 70919 71452 20645 63821 39469 45837 8597 10701 95589 66589 11373 46349 77947 80589 724 66934 87834 84782 64997 99368 18970 4127 72065 64916 59356 68563 74924 43545 63756 73124 56821 85225 77223 37107 82865 51363 98906 36212 34773 29275 92798 94226 97668 26361 53004 97367 81113 84355 93294 99194 91433 73260 53286 65796 24791 1242 25632 93969 11580 6649 9922 73023 32843 97068 44654 30432 76979 4227 72697 39657 63633 90035 9214 11835 26157 39093 85901 79006 97462 38446 27068 90623 99939 85056 2111 68856 79721 80469 80370 49412 75424 22883 48901 87910 74102 95806 82295 95426 43582 4424 3077 18612 29384 27895 58036 70879 330 96681 21273 28174 88091 66525 73683 50789 34247 56031 55384 18410 12023 4484 13857 58419 76459 83175 73031 58052 33854 23575 81172 90798 62311 81444 86483 55273 53235 85710 36052 99699 79576 70537 59658 26105 11058 51689 68491 10435 56162 56638 45445 73746 
34719 37879 12088 34323 49869 29515 47936 48834 20959 24336 46851 44297 17804 34426 29731 17433 46817 26013 49986 28330 47347 48307 31678 745 4708 39262 37048 34609 20406 47794 16618 27043 38833 12004 35027 46657 13424 42816 157 1010 46481 11716 25780 18930 104 13845 44150 11341 29121 44304 5273 38801 40687 42246 13909 8582 7879 18971 40571 26275 36854 10600 45112 49755 41519 18237 7358 33598 22710 20644 13805 4429 6038 8592 39280 48045 10991 24001 46747 2744 22290 27424 24486 24016 28621 37938 49343 3672 25989 1603 30458 3611 27316 3771 30104 13560 40466 12361 36065 38874 12676 32276 12772 27635 34544 18691 28158 17126 36474 43440 22104 34715 20223 49090 1513 8378 194 11869 37736 27307 878 38466 28192 45199 49127 23782 22409 1508 39573 23274 23338 4818 30996 17641 5444 11248 7301 31597 2175 33682 31465 42799 6199 47509 7165 17175 28198 3569 37963 46737 25752 20803 44715 12235 20766 7819 22063 9200 44864 7671 14808 12994 19977 48625 34340 29063 18831 9699 9221 46221 10073 22474 30557 13041 846 48315 27018 17813 48431 46411 23038 14539 25565 11335 8637 27337 15548 38707 45529 23735 38982 29214 41791 27653 42288 15742 18681 37542 37853 
197
71623 26384 19034 90844 9868 13709 87838 53558 40068 62697 67780 1501 4918 96870 57269 25347 43112 63473 553 11087 43455 52795 88978 80987 69818 43107 32303 39089 29252 76133 31285 58142 97662 94996 71418 36715 87656 69089 1976 49700 68947 691 23797 25443 33530 99246 56224 99400 14162 27248 5398 97742 18231 43203 82290 76567 30067 3637 70688 96829 20235 95714 50150 18014 18222 24786 87307 10647 66068 89679 13968 92628 92479 50680 4034 12102 52017 58342 38786 36550 2353 82700 97782 66536 54983 50216 19825 34633 88428 33278 98656 1364 74371 13883 78151 83309 72987 29992 66326 6811 26105 93686 99217 96391 36372 16298 30288 10585 60189 45621 38660 74705 91693 57302 56115 16443 80730 93482 88562 11881 80486 16139 15097 34330 1342 6925 88275 1804 47422 18573 31229 96244 37448 76183 25169 99853 46091 88157 42425 14103 12055 43862 38727 69836 44756 94920 4596 26715 69974 76269 24620 88139 56520 13197 69755 36640 99503 34728 37134 35993 50124 3067 85005 36970 10503 83085 36147 85979 81791 6021 63906 66705 48630 19456 7154 28760 9423 23042 92250 34149 79142 50991 49428 16739 84886 77476 27068 22712 44072 68514 38920 64739 68240 25900 45348 70624 51098 
32024 8614 19521 12420 6181 16556 7095 815 25873 24999 35502 7606 26706 2030 35183 44476 33001 38592 23624 25082 26546 15069 25397 15834 27560 19962 34260 19755 17254 31277 26017 48863 42959 9738 33962 7026 7200 11855 48367 8286 49779 32458 16093 29612 4006 32621 636 4804 598 39365 14640 23092 28340 41565 4710 22684 10707 13475 47331 35249 44913 28852 28947 10096 23599 12090 20095 48075 35298 45106 46512 6707 35455 42260 7769 42671 46160 10346 22879 24998 38593 23881 19599 17816 5834 34038 4523 6943 34949 8038 42370 37909 49146 24016 9204 35263 49825 5198 33805 21606 17034 31324 162 31079 513 35946 11260 19742 5923 17125 37788 440 31492 8899 15431 25643 39404 41300 32096 30894 23408 9353 19386 24859 3757 38662 9548 20775 46121 42682 27880 38523 711 45187 42442 31962 28405 27518 49323 41694 28149 8315 20218 39238 20011 9782 24554 42638 20447 7325 898 44373 13350 33818 31532 32476 29462 2311 46231 22316 7019 26487 22321 6588 16009 34480 7866 9890 42976 17167 43037 22182 34247 39298 18464 24357 2776 2920 36315 5539 43454 41647 958 36194 35829 2669 33632 42350 15596 14809 37173 6318 39773 18164 40269 24285 16045 
197
55440 78916 52963 90337 80651 11711 47969 1450 9133 38165 12544 44356 58945 94158 85205 6462 66361 58921 26525 57545 3613 44590 47382 52626 47046 96486 32852 11246 9769 39561 93853 73112 6230 65936 21882 9913 34266 22275 45445 82402 16751 29877 32547 99918 48983 24784 50220 97483 89703 48288 88716 91698 88995 93512 17439 92067 73970 726 43319 78575 17642 97935 60740 53961 53071 80757 13034 55529 2442 712 66521 24061 59892 66801 38267 85963 75065 62965 2643 87486 44124 62877 12106 248 55219 79960 6148 35942 1201 38332 23423 41494 98822 55320 79372 87508 28912 89550 77369 29682 85715 29936 7079 52427 41140 88134 31032 71265 43522 66307 4650 58610 95102 77053 28137 93878 7657 36250 79764 78907 6755 48241 36409 78650 93845 44752 52883 77581 45254 40410 88985 46190 61487 12630 2065 30829 10556 46056 15051 77100 73681 69299 2018 26241 65709 79877 50188 91237 69735 953 55397 99243 48874 28673 72031 86748 97469 8899 31872 34268 64438 30597 36673 7552 99223 622 47380 47425 34581 9153 83215 72239 63437 27868 20759 5284 28163 81862 7999 6382 12356 66879 63156 19329 36654 21947 95231 77205 55289 82496 53924 90787 34124 13266 25605 91442 46919 
2939 7949 27472 6108 45912 15410 30481 15409 26695 17780 30610 45137 46222 9746 44883 39969 4401 36994 28194 37075 7871 7716 27275 24462 14747 33939 2008 13094 27208 20299 32115 25921 24854 36662 26417 31973 28342 36359 31976 34156 30318 42526 30163 1484 30100 15900 16890 21416 3172 19247 36965 40756 9748 35096 46036 12407 13547 22880 47669 13838 36845 34303 20158 35273 16920 19806 29138 15765 12135 49072 33316 46474 29703 4724 30879 15422 32470 135 22161 420 13979 22359 42579 24146 14165 33772 13906 44652 14093 49845 27910 12564 26855 30763 13853 9822 45315 25757 7291 2384 5518 6067 38907 7994 29945 6574 43053 39142 26732 17728 37878 30552 36744 23332 37737 37656 19328 32124 6342 43057 12549 22544 25775 37325 7500 20469 16243 21068 32703 24613 5972 38175 1162 3657 7667 7086 49953 33349 44189 37555 14547 20860 46064 2938 33828 39844 19931 19027 4488 17861 42259 26669 38149 47297 42930 48916 46672 3184 3816 18767 26609 33049 32723 3920 12110 14711 45373 19730 27587 18595 32163 25273 17898 17926 25601 8731 5152 29967 41216 18671 618 49015 1924 14232 13917 45931 42065 27088 32914 10047 22014 29232 33884 46001 2988 30432 32451 
200
84150 38805 65606 40454 10074 7471 8386 72191 96031 52038 22560 35724 56069 43256 48682 77342 84475 71331 40047 52747 3597 33666 50612 67859 65858 77528 46756 21657 37432 59844 63871 87992 67267 27438 60790 2702 97496 3591 35890 50295 25490 57919 537 98771 94754 2798 6780 46496 30675 83243 76935 80460 39663 36958 62107 1918 54737 2546 14439 36444 79021 44804 86600 58631 24 7321 44998 34660 74065 21612 84190 48455 91652 85498 8792 98706 30570 59466 26660 86946 95026 18800 70108 69048 44148 16900 93892 48102 73749 26352 78577 60123 25332 26350 31688 40706 34854 65416 74925 6361 53282 57059 36615 43571 34633 9443 65699 91591 31958 30973 53333 30547 5366 73161 22284 95058 71533 15041 39173 29333 71224 95680 58995 82979 20231 97593 17880 43181 14532 39540 61485 2552 99502 63039 2994 63402 9281 88063 28689 84341 50978 87521 39763 96126 71467 95717 81981 51416 66643 51480 10916 81564 11869 50990 60830 57950 55065 35263 42482 56932 20878 14975 28604 89904 27903 66163 63282 19311 64153 63258 16135 65783 90532 48505 29264 81072 27908 79929 86362 81904 46115 36676 5692 93271 50758 71785 7290 14825 77058 37530 51945 18632 26076 69049 56112 87612 38989 86772 58388 9653 
29070 1547 6508 15165 26550 42086 35199 17934 7464 33445 25059 30305 11150 45545 37976 16226 5588 17226 19678 25020 41950 15349 48786 2789 30853 6155 22997 17651 19613 32580 46090 9092 28540 39473 46528 26648 13681 39140 19980 17813 31442 42081 16894 18510 26095 31155 25317 31357 43686 35138 10645 40195 30395 30390 21408 29476 15539 26163 29442 35984 14565 29390 25371 21700 30301 21376 44549 25133 30676 12570 11214 4904 2899 20237 12337 18686 35549 18874 36291 45716 28596 17115 20809 36451 24205 36040 48590 22272 14628 19253 37257 3140 3346 14965 35004 14509 48176 7448 36465 4626 43736 17041 45477 10082 3925 1233 48647 8967 34700 8211 35368 9863 35836 39910 39880 44656 38582 8006 13117 34469 37766 33556 29582 10649 7006 3627 18108 20124 11494 22018 542 4624 40478 28954 12434 32521 18332 30783 9656 20581 3223 19001 27415 11916 45766 28548 10985 40884 7350 45440 8470 30379 5346 20297 5393 13902 13849 11422 22645 27359 34082 15974 46143 23947 22987 38856 1780 35861 48557 6319 20205 8197 27008 753 28321 46201 2554 48729 3764 46703 39480 1997 40854 19529 22202 12073 47831 23078 2528 21324 29316 42286 84 43295 20183 19322 49230 40383 40764 20975 
198
17521 4665 2597 76949 22775 92039 5659 23449 71059 98252 22862 85773 54939 31617 1541 16629 69562 64675 61144 21748 60314 56303 44170 18503 14907 15752 48580 98605 33455 50517 71770 52002 19576 50377 16693 41167 83010 10681 59225 71020 71681 11108 83433 2473 46385 34609 84845 27183 86554 40840 98010 58215 53874 17661 33966 83569 58293 24207 43748 91085 98750 94186 5215 743 76893 57737 51871 2827 80428 64740 45224 60602 26048 99696 80731 86037 33355 85502 89323 23000 65835 47721 53192 29468 44126 85368 94581 41188 13662 43363 61916 83082 40880 57041 16661 51196 71482 72342 36106 35658 8740 1860 10074 95684 87502 72501 51889 74783 17019 32659 98258 75523 27019 40793 95649 45144 74165 49714 80626 35815 70259 60500 74438 93389 38126 28425 33730 88055 68976 18354 84378 24724 50324 15468 54393 68437 55164 95613 81503 62916 51087 8071 18064 13786 7775 86862 81304 180 60595 10642 35379 35033 5996 20494 44717 84897 14549 20800 69320 48537 74037 44731 26340 40377 71167 87676 90150 78971 73937 53173 1661 57670 33349 96250 66473 81523 96627 72488 15242 40360 17103 91225 46207 48843 79034 76179 11325 10846 82879 90729 83836 79510 21211 41187 75277 71043 83036 80822 
18838 417 24273 33369 39002 5450 19083 46592 22963 31210 27713 13451 30855 9729 36964 30877 2251 38230 22315 40678 25333 38679 2632 28547 33313 42481 49320 14186 35677 47950 32094 18788 14979 5687 1214 25605 26010 29827 38440 14602 4943 4788 4118 15178 29599 12794 43935 9360 13795 25699 42608 41452 49939 9826 33076 29530 19612 10897 12171 15399 41718 16027 20577 38875 16485 15651 23637 40461 2594 968 16849 5305 26322 5934 24604 4877 46285 23237 13218 12954 12573 24994 3874 14891 41169 47980 3916 47731 34766 24692 22184 4311 34800 36566 27535 15861 10201 49028 14933 21483 13189 12111 35291 22927 48419 40260 48089 47505 29799 30864 8199 37643 20595 4112 31905 25591 45851 23932 1647 13238 45608 6039 7903 23436 2088 42417 20171 32590 29833 25930 6406 43264 2201 24026 17120 26534 17044 38566 38278 20405 2142 36456 20524 37314 9843 5634 8698 44353 17022 44339 33088 43418 341 25388 39968 21167 33516 28875 12874 12300 4292 2331 1101 22817 40598 44064 5735 1419 45219 22000 4091 40234 7385 22167 16696 7953 47512 27207 45928 15420 23228 22060 17496 41824 28221 37678 26574 27273 34897 2257 43194 17484 23520 3161 45853 34733 8670 26483 
200
5237 62023 72955 33279 34230 51828 33284 80145 16296 67876 99120 77888 11124 12949 68536 32485 81882 20608 76960 975 81745 88177 95671 4774 50050 92389 36489 57713 46558 45701 66860 27082 55787 77247 17726 63112 95008 99044 44618 40508 90641 29587 10937 27207 75147 38857 19421 68547 16934 38897 1920 26388 4477 74858 65817 50544 20546 94340 59231 43642 62248 3773 56686 81969 97413 92210 94692 64198 77645 18514 34527 20179 50872 43398 76615 22545 15329 2126 34854 88521 33591 70762 18007 1438 93040 10933 44985 6466 39876 85485 8387 44448 9994 39128 58491 38249 95514 56739 80030 7038 26700 52241 84377 86875 45038 10712 23894 18731 93344 35843 90664 77251 64243 41442 45742 4807 29575 18892 95012 10628 62586 82149 47605 55925 87372 56885 80631 18338 23445 909 64635 81874 25774 98125 67581 86877 66985 16543 64156 93729 47727 82726 6763 4856 64413 56441 48067 28192 7561 60247 21853 89785 31922 74179 37651 376 75562 76891 92421 3945 79376 10695 57456 91975 47135 25873 65506 18990 4282 98426 25143 7424 72918 12006 22838 54484 33725 8232 76576 20883 10840 59166 90726 45874 67982 43436 42187 75279 15028 9322 69448 19164 3892 16172 82151 24218 13889 619 49883 63946 
48274 18925 35788 40307 17912 7702 33679 18320 24794 15903 22716 31512 14154 8404 31826 16870 26029 13516 47672 4856 23904 47202 37496 27018 9945 32415 20100 4718 42257 41583 31177 13628 479 780 29501 22343 14749 23879 43163 2530 29249 41220 24487 25793 3581 21106 44169 36449 18004 1801 37542 9255 34513 15096 35343 10747 37572 15324 45424 29985 46174 29673 24535 10377 34356 45230 26691 17572 38335 44450 38306 1630 26299 4225 17622 39121 13614 705 27802 6945 35571 30783 34942 20648 36471 5513 15323 15215 22722 27122 47890 26381 21837 28306 28431 42526 3225 15742 27198 2210 8183 2216 33440 30097 42349 40100 1294 4975 41664 18622 45023 14219 40963 34975 43526 23877 44085 20882 17630 29063 2128 39572 26000 48766 42520 3691 37207 48381 15202 4045 35370 15244 47231 14422 14967 27271 47595 15070 36031 38544 30349 4180 42895 47276 41293 27152 40387 8628 1008 15027 9013 47142 29450 41643 5092 14388 3235 34153 22093 44175 25308 29842 8976 48424 37410 10752 48917 40398 10837 4687 16517 15704 19198 18076 949 35791 47093 29412 13640 6628 7431 12424 46898 28443 6952 29933 10201 32922 11443 6250 46361 46447 7192 17625 32018 23503 49958 49150 23110 32284 
199
44708 79627 38303 38221 62645 28617 70995 54250 29485 34687 6086 38671 16845 43846 52826 11804 20823 18860 22035 43945 78321 317 62099 72697 16853 45064 24284 41275 11826 66069 84274 49362 56375 29403 12317 13787 20037 58698 99535 99884 85613 51151 39709 95424 60625 33070 1563 46781 29066 90828 74162 4929 47478 27832 90306 869 54532 5374 75055 82236 52638 99372 16824 45596 28106 31788 19029 55211 38568 25460 12176 88605 29535 17696 20429 89529 85403 47631 71776 57878 71076 74628 53504 41937 25633 64690 12009 46575 95281 60664 31661 85778 48686 14650 15061 7458 52609 7648 46512 91982 50821 78328 99026 2791 59740 31579 4270 49515 50510 72124 92132 60978 52814 38849 47893 63247 14040 32370 89368 97010 76852 88661 91077 64278 77632 40844 46938 96837 4518 4361 28621 33817 30099 12510 95986 6750 20493 13188 91345 25343 40105 7551 30342 62293 20965 68454 69974 85094 33725 99810 68160 59123 18942 43978 66683 39749 93970 18221 93403 54187 7626 84001 15585 46352 74539 5142 20888 15626 11978 87490 9967 52283 18578 41888 83988 43258 10881 99246 48783 81291 53173 93545 43326 89026 26551 2662 89916 53958 4767 74039 86278 8627 5482 90565 44639 6537 57444 56491 47462 
21401 25218 20838 31728 5942 7919 12290 6051 41927 15970 46154 30351 45829 3449 8431 25676 48413 35111 29734 22132 13965 29633 47931 42131 12230 35009 45885 20967 3768 1579 22401 6352 44322 25632 33504 22090 19174 9628 31914 49652 32673 33838 40376 31495 40174 20992 22573 42677 16536 13429 45111 16910 46980 13278 4173 14011 485 30981 29994 26402 45138 30663 27006 30578 2106 35857 953 45684 33507 31810 41913 4092 11189 47071 18239 49103 14928 8186 8211 7352 28444 10938 49536 49633 24170 4022 13000 25602 19601 25474 13063 45092 22392 6682 37019 17192 46422 49456 22426 38156 261 10104 22578 28450 42947 13037 37973 6043 36918 37892 4576 43954 47550 31529 16413 16273 10300 15067 22634 35256 26395 35853 902 49183 3053 23546 23767 15974 15122 6899 17536 13411 35233 30128 4177 30074 15110 77 12595 15862 8825 39181 46442 28119 31006 38416 27053 31434 13929 49958 47134 41718 37012 42206 44809 9149 18785 43789 19603 22522 1944 41033 49637 22913 10089 44753 24935 35271 7267 43514 336 16962 45778 25284 23038 35795 9458 33791 26903 20740 37404 32611 29908 39226 11385 4293 5406 45443 49526 1968 34804 24198 19577 29908 38344 42983 32693 30293 32351 
199
92422 94586 9073 61674 41159 43376 27854 51087 68754 10799 9832 96283 86104 45415 35676 71280 43923 73181 43521 38245 92833 54567 7264 93913 4399 41226 77482 62503 89764 18640 19223 22586 27212 36532 61862 72075 43686 59713 8199 59922 54658 32681 51299 34923 3431 43761 47233 46126 52132 70171 19124 98147 5603 15877 67483 63104 56812 20291 21949 45841 97999 43843 44684 35147 14506 31125 8826 78841 91614 51652 31019 70891 77014 50219 23646 79300 61399 56353 19628 69093 80206 47937 76472 59368 45280 37723 24423 41985 11950 67836 44270 12289 53319 80153 16925 44700 78415 53416 12809 72847 2890 43998 87472 54045 85482 95086 11887 40779 29531 94023 35104 56716 32727 89711 22747 50386 41386 49727 75762 39873 89829 89264 42696 15859 72968 77665 16255 12123 66182 67108 58104 64739 53079 38425 90154 23518 46083 84241 20789 45423 67719 85656 87224 15912 61759 34624 14278 88347 41531 34034 11201 18622 99129 13108 92365 58039 26713 39013 85635 21441 69135 80022 40321 78178 94592 59721 31921 52724 34504 14907 35366 6293 21155 33646 28824 38886 32375 65924 27163 42931 39094 82440 59489 16939 28776 39714 20932 94536 26740 10933 46591 763 96691 90324 14347 17052 29017 76791 18614 
17509 4721 33469 47986 30344 9376 42971 32059 49930 30245 6817 44408 31592 43670 37999 18041 9053 25967 11481 1510 23788 9726 3671 48937 24350 24852 18605 47293 37127 10329 8312 30260 25709 10496 4286 42425 34672 47135 6825 19924 5546 5721 31971 14075 40277 46890 11172 6106 19864 47099 47617 37986 39966 5978 4781 43107 8513 22166 36929 29467 30921 32740 32558 5189 20816 6657 1303 20335 28012 10662 34709 33437 48573 10738 4914 1273 25522 4531 38780 25754 3377 28646 45552 7079 3020 13196 46547 21029 14839 36965 26598 44536 45685 25102 7170 48440 42687 14279 37452 31044 10226 12082 37140 27422 24711 42565 42725 12926 47527 23850 46487 26658 9447 11312 34446 29168 49679 42782 42968 3760 6960 26516 42973 31467 12659 29830 1999 40033 2921 34060 33285 28722 7687 35544 25699 19147 23457 13984 15624 5933 18209 1073 39356 30237 6884 29260 7601 15629 11541 35813 23888 48547 233 13927 35201 24444 976 9385 49669 28945 2374 49491 9335 40979 15441 24520 38859 44508 38817 33264 20341 32405 10513 21408 5472 33046 6590 26308 26945 22724 4984 5390 28474 29560 39363 9585 32107 24852 32579 29648 42074 6245 49529 11271 40976 28168 36445 45924 30598 
197
9095 20181 54995 87768 48721 56448 17727 82361 10267 2278 47554 20934 71186 50838 70576 6330 18041 55342 61618 99543 63242 55111 88532 41111 24674 51032 96074 66463 4373 2026 66831 35483 12471 80990 58590 92228 24692 8822 92013 28669 16174 16885 51623 26599 92683 63220 17822 65729 94798 60278 17259 94721 42398 75007 23533 92056 76425 99239 12383 74682 52760 8318 70575 99846 18213 91011 76152 88642 79834 80187 22996 25954 26897 22007 91672 93873 68894 9711 25071 60554 3572 31649 14924 68810 2594 16605 2509 76059 44921 84943 12848 62923 76172 81190 43398 18215 89725 82048 95599 53342 94291 40541 89313 45841 39323 53532 917 1488 77322 31642 76749 33744 45353 71632 85188 4640 68333 49666 13969 84740 4177 57285 68274 87010 63684 37384 77924 82931 32809 36750 29675 44569 93991 36755 53233 91522 97822 29290 27274 77339 85818 47195 77035 56445 70287 74429 83425 70348 37577 3209 93918 81528 70597 40613 3696 50264 89649 73890 78093 33809 5810 55834 40268 4192 24628 74958 95072 98475 35426 23013 67831 1834 9712 30468 1922 42205 44911 12809 43589 48747 36508 53752 61949 23592 56019 82389 27749 42975 75910 2390 39888 27060 19590 85802 38427 24916 13813 
10864 23162 23542 38913 472 11562 44883 7559 32849 22512 24929 17738 11084 39230 5214 32145 35866 38732 28056 7114 49358 20240 4680 8807 5895 49961 41246 46108 24544 42979 23292 12944 20021 37620 31412 43656 28049 29138 42495 7707 21158 31263 34817 39223 20906 43019 26140 24740 45721 11305 26713 3839 30049 14274 1209 1369 5999 18042 8583 811 47972 26909 38502 39625 36716 15374 13990 46056 29226 42029 44697 43104 25748 40198 8241 29807 16671 27435 27547 21779 42309 20942 42539 20775 4924 46008 4800 42424 32608 22475 17759 45455 8953 18022 2160 36766 20398 23948 8951 34673 21343 37761 46772 49177 43288 19594 35631 21506 46429 46927 37126 4932 32662 23443 17020 34760 15440 16412 42785 39813 37904 39051 44087 9630 20354 13811 16124 5664 49425 45002 49180 13577 23138 43167 6400 26583 37659 21103 11928 16331 48903 32450 35663 34128 49279 23215 5786 46075 21726 29549 37704 7638 9930 35651 27299 26028 41943 10094 33920 6944 5294 8476 5117 21330 39205 586 48519 23168 8648 30538 4146 27646 39377 2569 33618 33103 42956 18127 47209 20510 21473 5040 46540 8559 18105 3599 44409 16310 6934 45063 11891 43324 39830 13266 39299 7514 10327 
