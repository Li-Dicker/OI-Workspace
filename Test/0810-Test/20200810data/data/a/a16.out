Case #1: 13977180
Case #2: 15107231
Case #3: 13591279
Case #4: 13978408
Case #5: 14054474
Case #6: 14163246
Case #7: 13789315
Case #8: 13858581
Case #9: 13876128
Case #10: 14317653
